src/renderers/native/ReactNativeMount.js

```diff
@@ -175,7 +175,7 @@ var ReactNativeMount = {
    * @param {View} view View tree image.
    * @param {number} containerViewID View to insert sub-view into.
    */
-  _mountImageIntoNode: function(mountImage, containerID) {
+  _mountImageIntoNode: function(mountImage : number, containerID : number) {
```

Comment:
Please don't add a space before the colon (we'll get linting setup at some point but until then…)